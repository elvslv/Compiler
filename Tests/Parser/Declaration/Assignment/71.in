type trec = record x: integer; end;
type tarr = array[1..10] of trec; 
function func: tarr; forward;
var i: integer;
a: trec;
begin
	a.x := func[10].y;
end.