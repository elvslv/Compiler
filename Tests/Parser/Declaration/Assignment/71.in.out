type TREC = record 
    var X : INTEGER;
end;
type TARR = array [const 1..const 10] of TREC;
function FUNC: TARR;
var I : INTEGER;
var A : TREC;

21 7 Unknown identifier in record access